pynamodb/attributes.py

```diff
@@ -478,6 +482,44 @@ def _instantiate(cls: Type[_ACT], attribute_values: Dict[str, Dict[str, Any]]) -
         AttributeContainer._container_deserialize(instance, attribute_values)
         return instance
 
+    def to_dynamodb_dict(self) -> Dict[str, Any]:
+        """
+        Returns the contents of this instance as a JSON-serializable mapping,
```

Comment:
What do you think about providing a really simple example of the output here to show the structure? Seems like a lot of people might trip up over the diff between dynamodb/normal. Would it also help to type as `dict[str, dict[str, Any]]`?